<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imodvObjedForm</class>
 <widget class="QWidget" name="imodvObjedForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>358</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>3dmodv Objects</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="colorFrame">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The color of the current object</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel1">
       <property name="text">
        <string>Object</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="objectSpinBox">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>See and enter current object</string>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel2">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="oneAllComboBox">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Select whether changes will be applied to only one object, all objects, ones that are on, or ones in group</string>
       </property>
       <item>
        <property name="text">
         <string>One</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>On's</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Group</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="objectSlider">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Select the current object</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the name of the object</string>
     </property>
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="checkBoxFrame">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Turn individual objects On/Off (use Edit-Object List for higher object #s)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="drawStyleComboBox">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Select whether data will be drawn as points, lines, or filled/lit regions</string>
         </property>
         <item>
          <property name="text">
           <string>Points</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lines</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fill</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fill outline</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel3">
         <property name="text">
          <string>Draw Data Type</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="dataTypeComboBox">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Select whether contour or mesh data will be drawn or turn drawing off or on</string>
         </property>
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contour</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mesh</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>On</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel4">
         <property name="text">
          <string>Drawing Style</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="Spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>49</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="meshingLabel1">
         <property name="text">
          <string>Meshing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="meshingLabel2">
         <property name="text">
          <string>Obj. 1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="panelListBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="panelFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="doneButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Close dialog box</string>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>32767</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Open help window</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>objectSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>objSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>objectSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>objectSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>objectSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneAllComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>editSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneAllComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataTypeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataTypeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>typeSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawStyleComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawStyleComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>styleSelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doneButton_2</sender>
   <signal>clicked()</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>donePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpButton_2</sender>
   <signal>clicked()</signal>
   <receiver>imodvObjedForm</receiver>
   <slot>helpPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
